schema/meter_provider.json

```diff
@@ -93,6 +93,9 @@
                 },
                 "without_scope_info": {
                     "type": "boolean"
+                },
+                "with_resource_as_constant_labels": {
```

Comment:
Ideally this would be named in the spec.. If we merge this can followup with a PR to the spec which proposes the same name. Not sure order matters.

Reply:
I just finished the PR for this in the Java SDK.
I was thinking more about a system property named `otel.exporter.prometheus.resource.attributes.add.regexp`; hence, can we call this field `add_resource_attributes_regexp`?

Reply:
This is now named in the spec as `with_resource_constant_labels`